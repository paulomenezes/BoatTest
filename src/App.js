import React from 'react';
import { View, Text, Image } from 'react-native';

import { Provider } from 'react-redux';

import Form from './components/Form';
import store from './configureStore';
import styles from './App.styles.js';

const App = props => (
  <Provider store={store}>
    <View style={styles.container}>
      <Text style={styles.intro}>
        Log-in now to view your upcoming and past flights
      </Text>
      <View style={styles.titleArea}>
        <Image
          style={styles.images}
          source={require('../assets/images/Image1.png')}
        />
        <Text style={styles.title}>Cut the queue. Check in now</Text>
        <Image
          style={styles.images}
          source={require('../assets/images/Image2.png')}
        />
      </View>
      <Text style={styles.description}>
        Check in online between 72 hours and 1 hour before departude. Head to
        the airport and get outta here. It's that simple!
      </Text>
      <Form />
    </View>
  </Provider>
);

export default App;
