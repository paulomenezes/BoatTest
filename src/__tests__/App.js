import 'react-native';
import React from 'react';
import App from '../App';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

jest.mock('../components/Form', () => 'Form');

it('renders correctly', () => {
  const tree = renderer.create(<App />);

  expect(tree).toMatchSnapshot();
});
