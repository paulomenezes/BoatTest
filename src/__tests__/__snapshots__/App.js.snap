// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "marginHorizontal": 10,
      "marginTop": 20,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "fontFamily": "OpenSans-Regular",
        "fontSize": 14,
        "marginTop": 10,
        "textAlign": "center",
      }
    }
  >
    Log-in now to view your upcoming and past flights
  </Text>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "space-between",
        "marginTop": 10,
      }
    }
  >
    <Image
      source={1}
      style={
        Object {
          "height": 40,
          "width": 40,
        }
      }
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Object {
          "fontFamily": "OpenSans-Bold",
          "fontSize": 16,
          "marginTop": 5,
          "textAlign": "center",
        }
      }
    >
      Cut the queue. Check in now
    </Text>
    <Image
      source={1}
      style={
        Object {
          "height": 40,
          "width": 40,
        }
      }
    />
  </View>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "fontFamily": "OpenSans-Regular",
        "fontSize": 14,
        "marginBottom": 10,
        "marginTop": 10,
      }
    }
  >
    Check in online between 72 hours and 1 hour before departude. Head to the airport and get outta here. It's that simple!
  </Text>
  <Form />
</View>
`;
